pdf/delivery-slip.product-tab.tpl

```diff
@@ -76,7 +76,7 @@
 						<td class="center"> &nbsp;</td>
 
 						<td>
-							{if isset($customization.datas[Product::CUSTOMIZE_TEXTFIELD]) && count($customization.datas[Product::CUSTOMIZE_TEXTFIELD]) > 0}
+							{if !empty($customization.datas[Product::CUSTOMIZE_TEXTFIELD])}
```

Comment:
Try `array_key_exists ( $key , $array )`
Also if this is a bug - it's best to target the correct branch.